### PR TITLE
add Beven Nyamande

### LINKED ISSUE #552: Beven Nyamande
image

### PR DESCRIPTION
**Pull request by bevennyamande**
resolve #552

### CODE REVIEW

**Comment by bevennyamande** [comment]
restarted and resolved.Thanks

File: data.json [hunk 1]
```diff
@@ -989,4 +989,11 @@
     "imageUrl": "https://user-images.githubusercontent.com/51391473/120854354-a57edc00-c542-11eb-910a-6dab08382357.png",
     "issueNumber": 550
   }
```
Comment (schmelto) [inline data.json:991]:
```suggestion
   }, 
```

File: data.json [hunk 2]
```diff
@@ -989,4 +989,11 @@
...
+  ,
```
Comment (schmelto) [inline data.json:992]:
```suggestion
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by bevennyamande** [comment]
done

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
Thanks!

**Comment by bevennyamande** [comment]
anytime

**Pull request merged**